rustic-compile.el

```diff
@@ -287,6 +288,9 @@ ARGS is a plist that affects how the process is run.
     (unless (plist-get args :no-display)
       (funcall rustic-compile-display-method buf))
     (with-current-buffer buf
+      (read-only-mode -1)
+      (insert (format "%s \n" (s-join " "  command)))
+      (read-only-mode 1)
```

Comment:
The idiomatic way of doing this would be using a `let` binding with `(inhibit-read-only nil)`

Reply:
Thanks, updated!